doc/content/the-things-stack/_index.md

```diff
@@ -27,15 +27,15 @@ The Things Stack V3 <a href="https://www.thethingsindustries.com/docs/reference/
 
 ## How can I access V3?
 
-The (free) community edition of The Things Stack V3 can be accessed via the <a href="https://console.cloud.thethings.network/" target="_blank">**Console**</a>. Make sure to select the cluster which is closest to you geographically.
+The (free) community edition of The Things Stack V3 can be accessed via the <a href="https://console.cloud.thethings.network/" target="_blank">**Console**</a>. Make sure to <a href="https://console.cloud.thethings.network/" target="blank_">select the cluster</a> which is closest to you geographically.
 
-[![The Things Network Console](TTN-V3-console.png "The Things Network Console")](https://console.cloud.thethings.network/)
+![The Things Network Console](TTN-V3-console.png "The Things Network Console")
```

Comment:
@benolayinka @laurensslats what is the final name here?

Reply:
@nejraselimovic The name changed from `The Things Network` to `The Things Stack Community`

Reply:
```suggestion
![The Things Stack Community](TTN-V3-console.png "The Things Stack Community Console")
```